test/functional/wallet_balance.py

```diff
@@ -55,7 +63,7 @@ def run_test(self):
         self.nodes[0].generate(1)
         self.sync_all()
         self.nodes[1].generate(1)
-        self.nodes[1].generatetoaddress(100, RANDOM_COINBASE_ADDRESS)
+        self.nodes[1].generatetoaddress(101, ADDRESS_WATCHONLY)
```

Comment:
The log above "Mining one block for each node" is now incorrect (or at least misleading). Can you change it?

Reply:
Huh, there are two calls to `generate(1)` for each node? How is it incorrect?

Reply:
Because there is then a call to generate to a watch-only address for node 1